tests/helper/test_helper.py

```diff
@@ -339,7 +339,7 @@ def test_display_error_if_present(
         case(
             {"type": "private", "to": [1], "content": "bar"},
             [["is", "private"]],
-            False,
+            True,
             id="all_private__pm__not_notified",
```

Comment:
We may need to update either the test or the ids, to represent what we're expecting here.

At the very least, this change is now inconsistent: True => notified :)

However, there are different aspects being notified here, so a boolean doesn't fully encapsulate the results any more.